site/content/en/guides/producer/setters/_index.md

```diff
@@ -87,6 +87,19 @@ spec:
   replicas: 3 # {"$kpt-set":"replicas"}
 ```
 
+#### Auto setters
+
+kpt auto-sets gcloud config setters when the package is fetched using `kpt pkg get`.
```

Comment:
Rephrase "kpt auto-sets gcloud config setters when the package is fetched using `kpt pkg get`."

It doesn't' really parse. Something along the lines of:

Some setters are automatically set when ...

Reply:
@frankfarzan Done.